src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -244,6 +264,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 		if (!this.model.activeCell || this.model.activeCell.id !== cell.id) {
 			this.model.updateActiveCell(cell);
+			this.model.activeCell.isEditMode = false;
```

Comment:
Is this correct? You update the active cell but then immediately set it to not be in edit mode? Is that because you don't want it to immediately start editing (since the editor will steal focus)? If so that seems like maybe `updateActiveCell` shouldn't be setting the edit mode then (or should have a param to control that at least)

Reply:
Can you comment on this?

Reply:
We don't want to set the editMode immediately after making the cell active to allow a intermediate state where the cell is only selected for keyboard navigation

Reply:
I think it would be better to change the updateActiveCell to take in an additional parameter then telling it whether to also set that cell in edit mode. That way others can call it and it keeps all the logic in one place.